src/Raven.Studio/typescript/commands/database/settings/enforceRevisionsConfigurationCommand.ts

```diff
@@ -0,0 +1,25 @@
+import commandBase = require("commands/commandBase");
+import database = require("models/resources/database");
+import endpoints = require("endpoints");
+
+class enforceRevisionsConfigurationCommand extends commandBase {
+
+    constructor(private db: database) {
+        super();
+    }
+
+    execute(): JQueryPromise<operationIdDto> {
+        const url = endpoints.databases.revisions.adminRevisionsConfigEnforce;       
+
+        return this.post<void>(url, null, this.db)
+            .fail((response: JQueryXHR) => {
+                this.reportError("Failed to enforce revisions configuration", response.responseText, response.statusText); 
+            })
+            .done(() => {
+                this.reportSuccess("Revisions configuration enforced successfully");
```

Comment:
please check if this is sync or async operation (in handler) - if async then we should write that config was enforced.

Reply:
`EnforceConfigRevisions` operation in handler is **sync** - so I assume it is ok like this.

Reply:
ok great - I was just double checking.